helm/vmss-prototype/templates/vmss-prototype.yaml

```diff
@@ -10,6 +10,35 @@
 {{- end -}}
 {{- end -}}
 
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kamino
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: node-updater
```

Comment:
I would name this related to kamino or even better, related to the job name.  (Same with the other elements)